README.md

```diff
@@ -32,24 +32,45 @@ There are 4 types of metric exposed:
 | -p        | --port             | Port on which to serve metrics                                                       |
 | -f        | --freq             | Frequency in hours with which to query the Prometheus API                            |
 | -r        | --regex            | If any found services don’t match the regex, they are ignored                        |
+| -a        | --auth             | Location of YAML file where Prometheus instance authorisation credentials can be found. For instances that don't appear in the file, it is assumed that no authorisation is required to access them. |
 
 ## Exposing Metrics
 
+### Dealing with auth'd Prometheus instances
+Some Prometheus instances will not let the exporter access the ```/api/v1/status/tsdb``` endpoint without providing some authorisation credentials. To access these instances, you must provide the authorisation credentials required. The solution to this depends on whether you are using the ```--proms``` or ```--service_discovery``` flag:
+- With ```--proms```: 
+    - Use the ```--auth``` flag to specify a YAML file mapping ```--proms``` instances to the values required.
+    - Example: \<my-prometheus\>:\<my-Authorization-header-value\>).
+- With ```--service_discovery```: 
+    - Use the ```--auth``` flag to specify a YAML file mapping instance identifiers to the values required.
+    - Identifiers can be at the namespace level, the Prometheus instance level, or the sharded instance level. 
+    - The naming convention is: \<namespace\>\[\_\<prometheus-instance-name\>\[\_\<sharded-instance-name\>\]\] (square brackets means optional). 
```

Comment:
Mind adding an example file for this? This isn't the clearest to read

Reply:
Done